engine-precompiles/src/native.rs

```diff
@@ -356,29 +386,46 @@ impl<I: IO> Precompile for ExitToNear<I> {
                 input = &input[32..];
 
                 validate_amount(amount)?;
+                let recipient = parse_recipient(input)?;
 
-                if let Ok(receiver_account_id) = AccountId::try_from(input) {
+                let (args, method, transfer_near_args) = if recipient.message
```

Comment:
We can get rid of the creation `String` here by using [as_deref](https://doc.rust-lang.org/std/option/enum.Option.html#method.as_deref) method.  
```rust
if recipient.message.as_deref() == Some(UNWRAP_WNEAR_MSG) {
...
```

Reply:
6e294387a84ee2e7fb386a905b1dca812f677c85